HARDWARE.md

```diff
@@ -181,7 +181,17 @@ Image date - 11-11-2023
 - Boots to desktop
 - UEFI panic in SETUP
 ```
+### Fujitsu
+- **Fujitsu Lifebook AH530**
+```
+Status - Booting
+Redox version - 0.8.0
+Variant - demo
+Image date - 11-11-2023
 
+- Displays desktop, mouse glide pad and buttons work correctly, Web browser window shows black inside borders, incorrect display dimensions offered at boot time, fan operating at max (does not happen with Ubuntu MATE), thermal shutdown.  Intel i5-460M cpu.
```

Comment:
Fans operating at max speed is because redox currently doesn't have any power management support built in. The ACPI driver isn't complete enough to support it. The firmware is supposed to automatically downclock the cpu when it gets too hot afaik, but the firmware on your system probably hasn't implemented this.